README.md

```diff
@@ -294,13 +294,13 @@ The web-app is able to automatically deploy the generated event sites to Github
 
 > Before making a pull request, please file an issue. So, other developers have the chance to give feedback or discuss details. Match every pull request with an issue please and add the issue number in description e.g. like "**Fixes #123**".
 
-**We have the following branches :-**
+**We have the following branches:-**
  * **development**
-	 All development goes on in this branch. If you're making a contribution,
+	 All development goes on in this branch. If you're contributing,
```

Comment:
What's the grammatical correction here? You just changed the way of saying the same thing, previous version is a completely valid english sentence